dockers/nvidia/Dockerfile

```
@@ -32,25 +32,24 @@ RUN \
         rm -rf pytorch-lightning ; \
         git clone https://github.com/PyTorchLightning/pytorch-lightning.git ; \
         cd pytorch-lightning ; \
-        git checkout ${LIGHTNING_VERSION} ; \
+        git checkout tags/${LIGHTNING_VERSION} ; \
```

Do we need this prefix `tags/`? It's fine if it works :)
```suggestion
        git checkout ${LIGHTNING_VERSION} ; \
```

---

I saw git complaining nut I ma fine with the simple way too